### PR TITLE
Add scripts, iframes, height, maxHeight, enterMode Options to documentation

### PR DESCRIPTION
**Pull request by alexander-schranz**

### CODE REVIEW

File: reference/content-types/text_editor.rst [hunk 1]
```diff
@@ -22,6 +22,21 @@ Parameters
...
+      - boolean
+      - Allow iframe tags in source code
+    * - height
+      - integer
+      - Set the text editor height
+    * - maxHeight
+      - integer
+      - Set the max height for autogrow '0' is  infinitely
+    * - enterMode
+      - string P/BR/DIV
```
Comment (dantleech) [inline reference/content-types/text_editor.rst]: I think this should simply be `enum`. The values should be given in the description.

File: reference/content-types/text_editor.rst [hunk 2]
```diff
@@ -22,6 +22,21 @@ Parameters
...
+      - Set what happens on enter allowed P, BR or DIV
```
Comment (dantleech) [inline reference/content-types/text_editor.rst]: you must put "code words" in double back-ticks. This both helps distinguish them from normal text (for humans) and also for the parser for when we add spell checking.

````
Set what happens, one of: ``p``, ``br`` or ``div``
````

**Comment by wachterjohannes** [comment]
@alexander-schranz ping

**Comment by alexander-schranz** [comment]
closed enter_mode ... are ckeditor specific parameters and example for ckeditor exist

**Pull request closed without merging**